README.md

```diff
@@ -15,7 +15,11 @@ If you're interested in participating in the Community Committee as a member, yo
 
 Community Committee meetings will be broadcast via Google Hangouts, will be announced ahead of time for access, and coordinated to optimize for contributor timezones.
 
-## Current Teams and Working Groups
+## Current Initiatives, Teams, and Working Groups
+
+### Initiatives
+Initiatives are projects that the Community Committee and members of the broader community are collaborating on to enable Node.js across the ecosystem.
```

Comment:
> Initiatives are projects

Would it be more appropriate to say:

"Initiatives are new projects..."

also:

> that the Community Committee and members of the broader community

"that the Community Committee and the broader community members ..."

Reply:
With regard to `new projects`, initiatives may be long-running projects under the Community Committee that may not fall under a subjective definition of "new". I think leaving it how it is will be most effective for the long-term accuracy of this section (reducing need for maintenance on it).

I will update the section with your second suggestion now 👍